autogpt_platform/backend/schema.prisma

```diff
@@ -206,16 +295,18 @@ model AgentGraphExecutionSchedule {
   lastUpdated DateTime @updatedAt
 
   // Link to User model
-  userId String
-  user   User   @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId        String
+  user          User         @relation(fields: [userId], references: [id], onDelete: Cascade)
+  AgentPreset   AgentPreset? @relation(fields: [agentPresetId], references: [id])
+  agentPresetId String?
 
   @@index([isEnabled])
 }
 
 model AnalyticsDetails {
   // PK uses gen_random_uuid() to allow the db inserts to happen outside of prisma
   // typical uuid() inserts are handled by prisma
-  id String @id @default(dbgenerated("gen_random_uuid()"))
```

Comment:
gen_random_uuid lets you insert with tools outside of prisma without having to generate your own uuid